ext/CreateParametersExt.jl

```diff
@@ -3,7 +3,7 @@ module CreateParametersExt
 import Thermodynamics.Parameters.ThermodynamicsParameters
 import CLIMAParameters as CP
 
-ThermodynamicsParameters(::Type{FT}) where {FT <: AbstractFloat} =
+ThermodynamicsParameters(::Type{FT}) where {FT <: Real} =
```

Comment:
Why not remove the restriction altogether? You can test that parameter structs are built with the right type in ClimaAtmos, it may not help much to enforce this here.

Reply:
I don’t think it makes sense to have a parameter struct with Ints.

Reply:
For sure only some types make sense, it's just you save yourself having to predict them in advance which might save you from having to write PRs like this (or eliminate the slowdown that happens when code doesn't work because of a wrong assumption like this, requiring source code update)